gwu-project-1/script.js

```diff
@@ -0,0 +1,15 @@
+let inputText = document.querySelector("#input-text");
+let addBtn = document.querySelector("#add-btn");
+let ol = document.querySelector("ol");
+
+let addName = () => {
+    let name = inputText.value
+    let li = document.createElement("li")
+    
+    li.innerText = name
+    ol.appendChild(li)
+
+    inputText.value=""
```

Comment:
Good job with resetting the input field after each entry. I would have preferred if you made this a separate function so that we can test it.

Something like:

```
const resetInputField = (input) => { input.value = '' }

// And then you can call the function in your addName method

const addName = () => {
   // other logic goes here
   resetInputField(input);
}
```